AIVEN_CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 ## Input Plugins
 
+### Elasticsearch
+
+* add cross cluster replication metrics ( they dont work for elasticsearch but its a first step unil we have an opensearch plugin )
```

Comment:
> they dont work for elasticsearch but its a first step unil we have an opensearch plugin

I'm not sure what this means, this should be expanded a bit more for clarity. Also typo in `unil`

Reply:
elasticsearch does not have this plugin api. It will only work for opensearch is what i mean

Reply:
fixed typo